models/repository.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import (
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/criteria"
+	"golang.org/x/net/context"
+)
+
+// WorkItemRepository encapsulates storage & retrieval of work items
+type WorkItemRepository interface {
```

Comment:
This interface already moved to `github.com/almighty/almighty-core/application` package, right?